ui/eureka/eureka.css

```diff
@@ -55,6 +55,13 @@
   padding-bottom: calc(1400% / 16);
 }
 
+.aspect-ratio-southhorn {
+  width: 100%;
+  height: 0;
+  position: relative;
+  padding-bottom: calc(1000% / 10);
```

Comment:
This `calc` operation is unneeded and this could just be `padding-bottom: 100%` I think.